10up/simple-local-avatars
# Fix/116 `admin.js` file merged into the main JS file.

### Description of the Change

- This PR fixes the issue of 404 to the `admin.js` file by moving the content from the `admin.js` file to the `assets/js/simple-local-avatars.js` file.
- Also removes the `assets/js/admin.js` file.
- The `/assets` folder was ignored in the .`distignore` file since #83, which prevented the `assets/js/admin.js` file to be compiled at the time of release.
- The code is already approved in #90 so it should be good to go asap.



Closes #116

### Alternate Designs

n/a

### Verification Process

1. Switch to the `develop` branch.
2. Go to `Settings > Discussion`.
3. Click `Clear cache`.
4. The page reloads, instead of AJAX should run.
5. Move to this PR’s branch `fix/116`.
6. Build the project by `npm run build`.
7. Reload the page and click `Clear cache`.
8. The AJAX should run.

### Checklist:






### Changelog Entry

> Fixed - Avatar cache not being cleared.

### Credits

Props @thefrosty @jeffpaul @faisal-alvi
